### DISCUSSION TITLE
Automate Query Row - Get newest row

### DISCUSSION THREAD

**Discussion by brandonhon** [Budibase/budibase#3469] [Help] [answered]
I am trying to use automate to query the rows and pull in the newest row created via the app. Currently I have a query rows action setup to use my table and sort on ascending on the auto id column. Currently no matter what I put in limit it returns all rows and not just the newest one created.

Any help would be appreciated. Thanks

**Comment by mjashanks** [comment, marked as answer]
Hi Brandon.

I have just tried this, exactly as you described. I set my limit to `1`, and it only pulled back one record. I was using a Budibase table. What kind data source are you using?

**Reply by brandonhon** [reply]
@mjashanks thanks for the quick reply. that was it. i was completely overthinking that and would have never guessed to just put that i only wanted '1' item returned. lol. i am using ta budibase table as well.

much appreciated. thanks bud.
